crates/integration-tests/dns-ptr.lua

```diff
@@ -0,0 +1,28 @@
+local kumo = require 'kumo'
+
+local function contains(list, t)
+    for key, value in pairs(list) do
+        if value == t then
+            return true
+        end
+    end
+    return false
+end
+
+-- check if we're able to resolve back to localhost
+local ok, a = pcall(kumo.dns.lookup_ptr, '127.0.0.1')
+assert(ok, 'expected localhost for 127.0.0.1 ptr')
+if ok then
+  assert(contains(a, 'localhost.'), 'expected localhost.')
+end
```

Comment:
The `assert(ok)` above means that we don't really need the `if` block around this:

```suggestion
assert(contains(a, 'localhost.'), 'expected localhost.')
```